### PR TITLE
HIP25: Validators

### PR DESCRIPTION
**Pull request by evanmcc**
This is an alternative approach to HIP 23, which we think is clearer and narrower, but in much the same spirit.

Ed: rendered view https://github.com/helium/HIP/blob/pevm/validators/0025-validators.md

### CODE REVIEW

File: 0025-validators.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# HIP 25: Miners
+
+- Author(s): @evanmcc, Helium Team
+- Start Date: 2021-01-11
+- Category: economic, technical
+- Original HIP PR: <!-- leave this empty; maintainer will fill in ID of this pull request -->
```
Comment (abhay) [inline 0025-validators.md:6]:
```suggestion
- Original HIP PR: [#110](https://github.com/helium/HIP/pull/110)
```

File: 0025-validators.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# HIP 25: Miners
```
Comment (abhay) [inline 0025-validators.md:1]:
```suggestion
# HIP 25: Validators
```

**Comment by abhay** [comment]
LGTM. Ready for discussion.

**Comment by mfalkvidd** [comment]
@evanmcc could you edit the link to the rendered view? It returns 404 now. https://github.com/helium/HIP/blob/e43ed8fae323db6dfbc8a71491670ed676710eec/0025-validators.md works.

**Pull request merged**